kitchen.docker.yml

```diff
@@ -28,27 +28,115 @@ platforms:
     driver:
       image: dokken/amazonlinux-2
     attributes:
+      ec2:
+        local_hostname: ip-10-0-0-200.eu-west-1.compute.internal
+        local_ipv4: 10.0.0.200
+        mac: 02:04:c9:66:6c:23
+        network_interfaces_macs:
+          02:04:c9:66:6c:23:
+            device_number: '0'
+            interface_id: eni-084e384b139c88340
+            ipv4-associations:
+              3.253.9.238: 10.0.0.200
+            local_hostname: ip-10-0-0-200.eu-west-1.compute.internal
+            local_ipv4s: 10.0.0.200
+            mac: 02:04:c9:66:6c:23
+            public_hostname: ec2-3-253-9-238.eu-west-1.compute.amazonaws.com
+            public_ipv4s: 3.253.9.238
+            security_group_ids: sg-0d6ed39436326b0c1
+            security_groups: Security Group kitchen
+            subnet_id: subnet-061d5d152f1ea6e1f
+            subnet_ipv4_cidr_block: 10.0.0.0/24
+            vpc_id: vpc-0f2e67ad1d48c13c6
+            vpc_ipv4_cidr_block: 10.0.0.0/16
+            vpc_ipv4_cidr_blocks: 10.0.0.0/16
```

Comment:
Should we add this block at `head_node_base` unit test level?
We should probably need it also for the validation step, which I don't see in the PR.
I would not set it up for every test on docker, unless you prove me wrong.

Reply:
These attributes are a mock of ec2 value. Every test in running on docker might leverage this configuration or override if needed. Do you prefer to keep this file cleaner moving the attributes under `head_node_base`?

I have tried to use the yaml references to not repeat the code but GITHUB does not liked it.